tests/e2e/backup_restore_suite_test.go

```
@@ -43,35 +43,32 @@ type ApplicationBackupRestoreCase struct {
+				return err
+			}
 		}
-		gomega.Eventually(lib.AreApplicationPodsRunning(kubernetesClientForSuiteRun, namespace), time.Minute*9, time.Second*5).Should(gomega.BeTrue())
-		err := lib.VerifyBackupRestoreData(runTimeClientForSuiteRun, kubernetesClientForSuiteRun, kubeConfig, artifact_dir, namespace, "restify", "restify", "restify", preBackupState, twoVol)
+
+		err := lib.VerifyBackupRestoreData(runTimeClientForSuiteRun, kubernetesClientForSuiteRun, kubeConfig, artifact_dir, namespace, routeName, appName, appName, preBackupState, twoVol)
 		return err
 	})
 }
 
+
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)